src/main/java/nl/tudelft/cse1110/andy/config/RunConfiguration.java

```diff
@@ -51,4 +52,8 @@ public Mode mode() {
         return Mode.PRACTICE;
     }
 
+    public ExternalProcess externalProcess() {
+        return null;
```

Comment:
One idea to avoid returning null here, and later having to check about nulls everywhere, is to simply create a `NoExternalProcess` class that implements this `ExternalProcess` interface. 

This class doesn't do anything. Simply pretends that the external process was executed. See? That would avoid having to do ifs. I implemented something somewhat similar in the Writer. See the `EmptyWriter` class.

This is called the Null Object Pattern: https://sourcemaking.com/design_patterns/null_object